session-1-jigsaw-intro/09_Automodules/README.md

```diff
@@ -0,0 +1,96 @@
+### External libraries example
+
+In this example we include external libraries, which haven't been modularized so far.
+These jar files can be translated into so called *"automatic modules"* by `javac`.
+To get an idea of how those modules will look and which dependencies they will require you can use the `jar` tool:
+
+    $ jar -d --file=lib/hamcrest-core-1.3.jar
+    o module descriptor found. Derived automatic module.
+    
+    module hamcrest.core@1.3 (automatic)
+      requires mandated java.base
+      contains org.hamcrest
+      contains org.hamcrest.core
+      contains org.hamcrest.internal
+and
+
+    $ jar -d --file=lib/junit-4.12.jar
+    No module descriptor found. Derived automatic module.
+    
+    module junit@4.12 (automatic)
+      requires mandated java.base
+       contains junit.extensions
+       contains junit.framework
+       contains junit.runner
+       contains junit.textui
+       contains org.junit
+       ...
+
+The outputs of those do sadly not tell us, that JUnit depends on Hamcrest.
```

Comment:
I think this whole block on `jdeps` is great and can be extracted into `jdeps.sh` like in other exercises so that we have the play around with the script and learn some more